Helper/Entity/ProductHelper.php

```diff
@@ -86,6 +89,25 @@ class ProductHelper
         'color',
     ];
 
+    /**
+     * ProductHelper constructor.
```

Comment:
Hi @peterjaap !
Looks that there's a separation line missing between the description of the constructor and the first param (it breaks the quality-tools check).
Could you please add it and then I think we're good to merge ;)